pkg/csi/service/wcp/controller_helper.go

```diff
@@ -165,10 +165,10 @@ func validateWCPControllerExpandVolumeRequest(ctx context.Context, req *csi.Cont
 			return logger.LogNewErrorCodef(log, codes.Internal,
 				"failed to get config with error: %+v", err)
 		}
-		vmOperatorClient, err := k8s.NewClientForGroup(ctx, cfg, vmoperatorv1alpha5.GroupName)
+		vmOperatorClient, err := k8s.NewClientForGroup(ctx, cfg, vmoperatorv1alpha2.GroupName)
```

Comment:
This is really confusing to me. vmoperatorv1alpha2 or any vmoperatorv1alphaX all have the same group name. Create a constant and use it consistently throughout.